stubs/PySocks/sockshandler.pyi

```diff
@@ -0,0 +1,57 @@
+import http.client
+import urllib.request
+from collections.abc import Mapping
+from typing import Any, TypeVar
+from typing_extensions import override
+
+import socks
+
+_T = TypeVar("_T")
+
+def merge_dict(a: Mapping[_T, Any], b: Mapping[_T, Any]) -> dict[_T, Any]: ...  # undocumented
```

Comment:
I'm not sure about using `Mapping` because it doesn't have `copy` method.
and based on `a` is `dict` and `update` builtin method (where `_K`,`_V` - TypeVars):
```suggestion
def merge_dict(a: dict[_K, _V], b: SupportsKeysAndGetItem[_K, _V]) -> dict[_K, _V]: ...  # undocumented
```

Reply:
Done.